src/Service/CopernicaRestApiService.php

```diff
@@ -39,4 +39,16 @@ public function createProfile(int $databaseNumber, array $data)
     {
         return $this->post(sprintf('database/%d/profiles', $databaseNumber), $data);
     }
+
+    /**
+     * @param int   $databaseNumber
+     * @param array $data
+     * @param array $parameters
+     *
+     * @return mixed
+     */
+    public function putProfile(int $databaseNumber, array $data, $parameters)
```

Comment:
Since (according to the docblock) the `$parameters` parameter is also a array we should add the type-hinting here